### ISSUE TITLE
Support collab element in person-group

### ISSUE THREAD

**Issue by michael** [substance/texture#423]
We currently can't deal with collab elements in person groups. It is important to add that, as it is limiting the ability to build a bibliography in the editor.

Such things should be possible:

```xml
<ref id="bib28">
<element-citation publication-type="book">
<person-group person-group-type="author">
<collab>National Research Council</collab>
</person-group>
<year iso-8601-date="2003">2003</year>
<source>
Guidelines for the Care and Use of Mammals in Neuroscience and Behavioral Research
</source>
<publisher-loc>Washington, D.C</publisher-loc>
<publisher-name>National Academies Press</publisher-name>
<pub-id pub-id-type="doi">10.17226/10732</pub-id>
</element-citation>
</ref>
```

The current workaround is to tag organisations (groups) as persons (`name` element).

@Melissa37 at eLife can `collab` elements occur in each publication-type or just in some?

**Comment by Melissa37** [comment]
Hi there

We need them to be allowed for all publication types.

Thanks
M

**Comment by michael** [comment]
Solved via #588.
